src/main/webapp/possibleInterviewers.jsp

```diff
@@ -0,0 +1,38 @@
+<%@ page import="java.util.Set" %>
+<%@ page import="com.google.sps.data.PossibleInterviewer" %>
+<%
+  Set<PossibleInterviewer> set = (Set<PossibleInterviewer>) request.getAttribute("interviewers");
+  pageContext.setAttribute("set", set);
+  String utc = request.getParameter("utcStartTime");
```

Comment:
Any reason not to inline all of these into pageContext.setAttribute?

Reply:
Inlined.